pkg/controllers/managementuser/resourcequota/resource_quota_sync.go

```diff
@@ -244,39 +239,29 @@ func (c *SyncController) getExistingLimitRange(ns *corev1.Namespace) (*corev1.Li
 	return limitRanger[0], nil
 }
 
-func (c *SyncController) getNamespaceResourceQuota(ns *corev1.Namespace) (*corev1.ResourceQuotaSpec, error) {
-	limit, err := getNamespaceResourceQuotaLimit(ns)
+func (c *SyncController) getRequestedResourceQuotaLimit(ns *corev1.Namespace) (*v32.ResourceQuotaLimit, error) {
```

Comment:
I think this would look fine, given that we also return the K8s quota spec:
`requestedQuotaLimit, newQuotaSpec, err := c.deriveRequestedResourceQuota(ns)`